examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/generate-entity.ts

```diff
@@ -69,13 +69,14 @@ export const generateEntity = async (
     const measureUnitsService = new MeasureUnitsService(
       serviceConstructorProps
     );
-    const measureUnits = await measureUnitsService.create();
+    await measureUnitsService.create();
```

Comment:
What is the point of creating measure units but after that fetches them again?
`create` will create `N` measure units to use them. But as I can see you create them but then fetch them again `.getAll()`. Is there any reason for that?

Reply:
If an Entity already has measure units with the same names, the creation will not go any further. Sometimes it is convenient to run Invoice generation multiple times.

Reply:
Hm, maybe then we should run `measureUnitsService.create()` only once to avoid any duplicates and provide that structure to `generate-entity` and reuse created values. Or as you did: create it once but inside `generate-entity` just call `getAll()` and take a few to add it to invoices

Anyway it's not so important